services/install-turtleblocksjs.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+function install {
+  # create service directory
+  mkdir -p /srv/turtleblocksjs
+
+  # create yml(s)
+  {
+    echo "version: '3'"
+    echo "services:"
+    echo "  turtleblocksjs:"
+    echo "    image: treehouses/turtleblocksjs:balena"
```

Comment:
balena is not a good tag ...

Reply:
what tag will be better?
am I supposed to add comments about arm6 and arm7 on it?

Reply:
```
root@lilyxxx0w:~/cli# ./cli.sh services available
couchdb
kolibri
mariadb
mastodon
moodle
netdata
nextcloud
ntopng
pihole
planet
portainer
privatebin
seafile
turtleblocksjs
```
for the pi0w (arm6), services available works.

Reply:
also works on arm7
```
pi@treehouses:~/git_repo/cli $ sudo ./cli.sh services available
couchdb
kolibri
mariadb
mastodon
moodle
netdata
nextcloud
ntopng
pihole
planet
portainer
privatebin
seafile
turtleblocksjs
```